plugins/modules/aws_acm.py

```diff
@@ -193,6 +193,22 @@
     state: absent
     region: ap-southeast-2
 
+- name: request a certificate issued by ACM
+  community.aws.aws_acm:
+    certificate_request:
+      domain_name: acm.ansible.com
+      subject_alternative_names:
+      - acm-east.ansible.com
+      - acm-west.ansible.com
+      validation-method: DNS
+      idempotency-token: "91adc45q"
```

Comment:
Notes: Idempotency tokens time out after one hour. Therefore, if you call RequestCertificate multiple times with the same idempotency token within one hour, ACM recognizes that you are requesting only one certificate and will issue only one. If you change the idempotency token for each call, ACM recognizes that you are requesting multiple certificates.